src/proto/FunctionRpc.proto

```diff
@@ -179,24 +173,25 @@ message FileChangeEventRequest {
 
 // Indicates whether worker reloaded successfully or needs a restart
 message WorkerActionResponse {
-  // indicates whether a restart is needed, or reload succesfully
+  // indicates whether a restart is needed, or reload successfully
   enum Action {
     Restart = 0;
     Reload = 1;
   }
-  
+
   // action for this response
   Action action = 1;
 
   // text reason for the response
   string reason = 2;
 }
 
-// NOT USED
-message WorkerStatusRequest{
+// Used by the host to determine worker health
+message WorkerStatusRequest {
 }
 
-// NOT USED
+// Worker responds with status message
+// TODO: Add any worker relevant status to response
```

Comment:
same as above

Reply:
still relevant, this todo is part of the dynamic concurrency feature